templates/plate-playground-template/src/lib/plate/plate-plugins.ts

```diff
@@ -154,6 +154,7 @@ import { TableRowElement } from '@/components/plate-ui/table-row-element';
 import { TodoListElement } from '@/components/plate-ui/todo-list-element';
 import { withDraggables } from '@/components/plate-ui/with-draggables';
 import { TabbableElement } from '@/components/tabbable-element';
+import { createDeletePlugin } from '@udecode/plate-select';
```

Comment:
Please undo changes on this template. We generally update the templates after publishing.